run/run.py

```diff
@@ -176,6 +176,12 @@ def main(platform_id, platform, args, config):
                 command.insert(5, args.path)
             if platform['browser_name'] == 'chrome':
                 command.extend(['--binary', browser_binary])
+
+                # temporary fix to allow WebRTC tests to call getUserMedia
+                command.extend(
+                    ['--binary-arg=--use-fake-ui-for-media-stream'])
+                command.extend(
+                    ['--binary-arg=--use-fake-device-for-media-stream'])
```

Comment:
These can be combined

```
command.extend([
    '--binary-arg=--use-fake-ui-for-media-stream',
    '--binary-arg=--use-fake-device-for-media-stream'
])
```

Reply:
Ping @kereliuk to combine these so we're all set for Monday.